projects/ssr-tests/src/ssr-testing_spec.ts

```diff
@@ -0,0 +1,85 @@
+import * as Log from './log.utils';
+import * as ProxyServer from './proxy.utils';
+import * as Ssr from './ssr.utils';
+
+const BACKEND_BASE_URL: string = process.env.CX_BASE_URL || '';
+
+describe('SSR E2E', () => {
+  let proxy: any;
+  const REQUEST_PATH = '/electronics-spa/en/USD/';
+
+  beforeAll(async () => {
+    Log.clearSsrLogFile();
+    await Ssr.startSsrServer();
+  });
```

Comment:
[MEDIUM]
I believe, my intention was to start SSR server separately for _each_ spec (not have one shared SSR instance for all specs - because it's prone to specs interfering with each other, e.g. because some leftover side-effect behaviours after the previous request, that might affect the next request's handling).